### PR TITLE
added manage  Insider Preview builds org link

### PR DESCRIPTION
**Pull request by RAJU2529**
this is my own PR, i added  Manage Insider Preview builds across your organization link and its explanation  (not own explanation ) I copied and pasted from below article
**https://docs.microsoft.com/windows-insider/business/manage-builds**
I need help from @JohanFreelancer9 to verify . So please assist me with your suggestions.

Thanking you

### CODE REVIEW

**Comment by RAJU2529** [comment]
@JohanFreelancer9 , Please verify this PR #9841,  also make sure added sentences are correct and essentially required to this article.

File: windows/whats-new/windows-11.md [hunk 1]
```diff
@@ -47,6 +47,8 @@ For more information about device eligibility, see [Windows 11 requirements](win
 
 If you are interested in testing Windows 11 before general availability, you can join the [Windows Insider Program](https://insider.windows.com) or [Windows Insider Program for Business](https://insider.windows.com/for-business). You can also preview Windows 11 by enabling pre-release Windows 10 feature updates in [Microsoft Endpoint Configuration Manager](/mem/configmgr/core/servers/manage/pre-release-features) or [Windows Server Update Services](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/publishing-pre-release-windows-10-feature-updates-to-wsus/ba-p/845054) (WSUS).
 
+If you're an administrator, you can manage installations of Windows 11 Insider Preview Builds across multiple devices in your organization using Group Policy, MDM solutions such as Intune, Configuration Manager, or [Windows Server Update Services](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/publishing-pre-release-windows-10-feature-updates-to-wsus/ba-p/845054) (WSUS).For more informinformation see [Manage Insider Preview builds across your organization](https://docs.microsoft.com/windows-insider/business/manage-builds).
```
Comment (JohanFreelancer9) [inline windows/whats-new/windows-11.md:50]:
```suggestion
If you are an administrator, you can manage installations of Windows 11 Insider Preview Builds across multiple devices in your organization using Group Policy, MDM solutions such as Intune, Configuration Manager, or [Windows Server Update Services](https://techcommunity.microsoft.com/t5/windows-it-pro-blog/publishing-pre-release-windows-10-feature-updates-to-wsus/ba-p/845054) (WSUS). For more information, see [Manage Insider Preview builds across your organization](/windows-insider/business/manage-builds).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RAJU2529, looks good, just a few edits, please. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by RAJU2529** [comment]
Still no updates for #9841

**Review by greg-lindsay — APPROVED** [review]

**Comment by RAJU2529** [comment]
@greg-lindsay Thanking you sir.

**Pull request merged**